### ISSUE TITLE
Tradables list not showing

### ISSUE THREAD

**Issue by kaioloureiro** [Revadike/EnhancedBarter#51]
My tradables are totally blank when I try to begin an automated offer. Trading Cards are the only items show.

**Comment by Revadike** [comment]
Do you have any screenshots? Also, what are your settings at https://barter.vg/u/my/? Finally, what do you see when you send a multi-user offer?

**Comment by kaioloureiro** [comment]
Which settings you're talking about? I think they're all fine. My multi-user offer page seems to be fine too, I can see my tradables there but isn't what I want to use for now.

Below you can see my automated offer page. I'm having this issue for a couple months but waited to see if was some bug on my side but now I think it's not.
![barter](https://github.com/Revadike/EnhancedBarter/assets/22198551/e90167fb-72c0-4e72-beb9-3dd659418509)

**Comment by Revadike** [comment]
Check this: `[ send to no specific user ]`

**Comment by kaioloureiro** [comment]
> 

Sorry, it says "[ send to no specific user ]" to me. I can't see what you sent.

**Comment by Revadike** [comment]
> > 
> 
> Sorry, it says "[ send to no specific user ]" to me. I can't see what you sent.

Do this and show me the result:
![image](https://github.com/Revadike/EnhancedBarter/assets/4411977/fc55f0ed-f748-42b7-b331-576d344644c5)

**Comment by kaioloureiro** [comment]
Same as when creating to multi user offer. My tradables are shown, but it not serve to my purposes. I don't want to trade my games to specific games from my wishlist, nor create any offer manually. I just want to do what the automated offer should do. This is the only page where my tradables are not available.
![barter2](https://github.com/Revadike/EnhancedBarter/assets/22198551/9c217c98-788a-438e-ad8d-7b27233bab59)

**Comment by Revadike** [comment]
Idk, I can't reproduce it.

**Comment by kaioloureiro** [comment]
> Idk, I can't reproduce it.

Yeah, I really don't know what happened too. Maybe it's because I have too many games on my library? Idk.
